ui/src/views/infra/AddPrimaryStorage.vue

```diff
@@ -879,7 +879,7 @@ export default {
           url = values.flashArrayURL
         }
 
-        if (values.provider === 'Linstor') {
+        if (values.provider === 'Linstor' || params.provider === 'Linstor') {
```

Comment:
given line 788 and also lines 868-869 , how is this making a difference?

Reply:
Because if I pick the protocol the `values.provider` isn't changed, and would be whatever the default is.
Thats why I set the `params.provider` to `Linstor` and check for it afterwards

Reply:
ok, it seems 
```suggestion
        if (values.provider === 'Linstor' || values.protocol === 'Linstor') {
```
would cut out the middle man, but lgtm

Reply:
Guess would also work, but I have to set the `params.provider` anyway. I'll test your suggesten tomorrow

Reply:
@rp- update on the test, and confirm the final changes. is this critical and to be included in 4.19.1.0?

Reply:
> @rp- update on the test, and confirm the final changes. is this critical and to be included in 4.19.1.0?

Just tested and daans suggestion works too.
Yes this is critical as it is a regression.